.github/workflows/main.yml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches: "*"
   push:
+    branches: "master"
     tags: "*"
```

Comment:
@cclauss this combo avoids double CI builds on PR's for both `pull_request` and `push` events, and runs CI/CD when tags are pushed to master 👍

Reply:
So if I push a pull request to the `install` branch then no CI is run?

Reply:
Nice!!